### PR TITLE
Cache view request args

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: flask_resty/authentication.py [hunk 1]
```diff
@@ -2,17 +2,13 @@
 
 # -----------------------------------------------------------------------------
 
-REQUEST_CREDENTIALS_KEY = 'request_credentials'
```
Comment (itajaja) [inline flask_resty/authentication.py]: why remove this?

Comment (taion) [reply flask_resty/authentication.py]: idk, do you think having this random constant is that useful? i was looking at it more and thought it added more noise than anything else

**Comment by taion** [comment]
@itajaja normally we just call `self.get_request_args()` with no kwargs set. i don't think we ever pass in any kwargs. since the kwargs themselves aren't nicely hashable, it seems like a good trade-off for performance and complexity to only handle the "normal" case.

**Comment by itajaja** [comment]
fair enough. what about adding `request_args` property that calls into `get_request_args` then? I think it's clearer that's a memoized value

**Pull request merged**